# HG changeset patch
# User Rik <rik@octave.org>
# Date 1344977864 25200
#      Tue Aug 14 13:57:44 2012 -0700
# Node ID c7d466a02065fc54aa4c62dc58bd44a3d314652b
# Parent  7a19e8275d41f634c6bc3504eb20c8ade78b434b
configure.ac: Add checks for a few missing types.

configure.ac: Add AC_TYPE_INT64_T, AC_TYPE_SSIZE_T,
AC_TYPE_UINT64_T, AC_CHECK_TYPES([ptrdiff_t]).

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -1813,23 +1813,27 @@ fi
 
 ## For MSVC compilers, avoid #define of min/max from windows.h header
 if test "$have_msvc" = "yes"; then
   AC_DEFINE(NOMINMAX, 1, [Define to 1 if you want to avoid min/max macro definition in Windows headers.])
 fi
 
 ### Determine types and size of types.
 
+AC_TYPE_INT64_T
 AC_TYPE_MODE_T
 AC_TYPE_OFF_T
 AC_TYPE_PID_T
 AC_TYPE_SIZE_T
+AC_TYPE_SSIZE_T
 AC_TYPE_UID_T
+AC_TYPE_UINT64_T
 AC_CHECK_TYPES([dev_t, ino_t])
 AC_CHECK_TYPES([long long int, unsigned long long int])
+AC_CHECK_TYPES([ptrdiff_t])
 
 ## How big are ints and how are they oriented?
 ## These could probably be eliminated in favor of run-time checks.
 
 AC_CHECK_SIZEOF(short)
 AC_CHECK_SIZEOF(int)
 AC_CHECK_SIZEOF(long)
 AC_CHECK_SIZEOF(long long)
